services/crypto/key/features.yaml

```diff
@@ -0,0 +1,103 @@
+common_features:
+  - CCC.F01 # Encryption in Transit Enabled by Default
+  - CCC.F02 # Encryption at Rest Enabled by Default
+  - CCC.F03 # Access Logs
+  - CCC.F06 # Identity Based Access Control
+  - CCC.F07 # Event Notifications
+  - CCC.F08 # Multi-zone Deployability
+  - CCC.F09 # Monitoring
+  - CCC.F10 # Logging
+  - CCC.F13 # Infrastructure as Code
+  - CCC.F14 # API Access
+features:
+  - id: CCC.KeyMgmt.F01
+    title: AES-256
+    description: |
+      Support for the AES-256 Advanced Encryption Standard with a 256-bit key for encryption and decryption.
+      Note: Azure only supports symmetric keys in the managed HSM service.
```

Comment:
I think note is bit misleading here. For me it will make more sense if it was written as 
Note: AWS and Google Cloud support this algorithm by default, whereas Azure's support for this is limited to its managed HSM (Hardware Security Module) service.

Reply:
We probably don't want to make point-in-time assertions about CSP features, right?